### PR TITLE
fix(streaming): Fix stream SSE, not necessary close stream.

### PR DESCRIPTION
**Pull request by damianpumar**
### Author should do the followings, if applicable

- [X] Add tests
- [X] Run tests
- [x] `yarn denoify` to generate files for Deno

Related to this issue: https://github.com/honojs/hono/issues/2319

### CODE REVIEW

**Comment by yusukebe** [comment]
Hi @damianpumar, Thanks!

@watany-dev Could you review this?

File: src/helper/streaming/sse.ts [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import type { Context } from '../../context'
 import { StreamingApi } from '../../utils/stream'
+import { stream } from '.'
```
Comment (watany-dev) [inline src/helper/streaming/sse.ts:3]: This will surely be unnecessary.

**Comment by watany-dev** [comment]
@damianpumar 
Thanks, I understand the intent of the fix, including the issue. I made a small comment, please take a look. Also, when it's all done, a `bun run denoify` would eliminate the CI error.

File: src/helper/streaming/sse.ts (hunk 1 above)
Comment (damianpumar) [reply src/helper/streaming/sse.ts:3]: You're right!.
Thanks for catch it!

**Comment by yusukebe** [comment]
@damianpumar Thanks!

@watany-dev Can you review this again? It looks good to me.

**Comment by yusukebe** [comment]
Thanks! Merge now.

**Comment by rohanrehman** [comment]
@yusukebe thanks for the flame ,@damianpumar thanks for the fix, I see it's merged into main. 
Currently testing hono@4.6.3, are the fixes in this release?

**Pull request merged**